src/test/librados/aio_cxx.cc

```diff
@@ -2338,3 +2338,41 @@ TEST(LibRadosAio, PoolEIOFlag) {
   ASSERT_TRUE(max_success + 1 == min_failed);
   my_lock.unlock();
 }
+
+TEST(LibRadosAio, MultiReads) {
```

Comment:
i like the idea of this test, but we really want it to test the actual code in `SimpleRADOSStriper::read()` instead of a copy of its logic. that way if we change `SimpleRADOSStriper` in the future, this test can catch issues there

@batrick is `test/libcephsqlite/main.cc` the right place for this?